packages/ember-metal/lib/tags.js

```diff
@@ -19,11 +20,19 @@ export function tagFor(object, _meta) {
   }
 }
 
+function K() {}
+function ensureRunloop() {
+  if (!run.backburner.currentInstance) {
+    run.backburner.schedule('actions', K);
+  }
+}
+
 if (hasGlimmer) {
   ({ DirtyableTag, CONSTANT_TAG, CURRENT_TAG } = require('glimmer-reference'));
   makeTag = function() { return new DirtyableTag(); };
 
   markObjectAsDirty = function(meta) {
+    ensureRunloop();
```

Comment:
since the current ember-glimmer test harness always starts a run loop with runTask() we could do this in a followup PR, just focus on getting the other part in place and validating the tag.